demos/twa-notification-delegation/src/main/AndroidManifest.xml

```diff
@@ -29,7 +29,7 @@
             android:label="@string/app_name"
             android:exported="true">
             <meta-data android:name="android.support.customtabs.trusted.DEFAULT_URL"
-                android:value="https://twa-tests.firebaseapp.com" />
+                android:value="https://mvano.github.io/notifications/index.html" />
```

Comment:
Did you intend to do this? Provided you're going to keep your site up (and it's reasonably self-explanatory) I don't mind leaving it in.

Reply:
Undone.